src/ucp/core/ucp_context.c

```diff
@@ -522,6 +527,11 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "Prefetch memory when using global VA MR",
    ucs_offsetof(ucp_context_config_t, gva_prefetch), UCS_CONFIG_TYPE_BOOL},
 
+  {"FORCE_REQUEST_FLAGS", "none",
```

Comment:
Have you tested it manually, it seems like there should be empty default value instead of "none". 

```suggestion
  {"FORCE_REQUEST_FLAGS", "",
```

Also I tried to set `UCX_FORCE_REQUEST_FLAGS=fast_cmpl` and get the following issue:

![image](https://github.com/user-attachments/assets/f6eb3e8b-74b8-4ceb-9b2e-1bd82ca56919)

Reply:
Changed to "auto"
![image](https://github.com/user-attachments/assets/d32fa74e-e9c9-4812-b04e-d4d4dd270b16)